.eslintrc

```diff
@@ -1,7 +1,14 @@
 ecmaFeatures:
   experimentalObjectRestSpread: true
 
+plugins:
+  - meteor
+
+parser: babel-eslint
+
 rules:
+  strict: 0
+  no-undef: 2
```

Comment:
Enabling `no-undef`, so ESLint warns when undefined variables are used. This works in conjunction with `meteor/globals`, which defines Meteor's global variables (like `Meteor` and `Session`) depending on the environment of the file (`/client`, `/server`, or no special folder).

So `Session` will be defined in a file in `/client`, but not in a file in `/server`. In the latter case `no-undef` will warn when accessing  `Session`.